handler_wrapper.go

```diff
@@ -84,6 +85,7 @@ func (hw *HandlerWrapper) Invoke(ctx context.Context, payload interface{}) (resp
 		// We're within the timeout window
 		case <-timeoutChannel:
 			hw.Log.Debug("Function is about to timeout, sending report")
+			hw.Label("@iopipe/timeout")
```

Comment:
If there is a timeout, does an error also occur?

Reply:
@XerTheSquirrel We don't treat timeouts the same as other errors because a timeout is the agent interrupting execution and not an error the user has introduced. Currently the Python agent only labels with `@iopipe/timeout` in a timeout situation to reduce noise. This way if the user is actually looking for timeouts, they can find them. And if their looking for errors they can find them. But `@iopipe/error` won't find both. If you think this should work differently, feel free to add an issue to the next throwdown and we can discuss further.